### DISCUSSION TITLE
My NAS fails to remount after I reboot my pi

### DISCUSSION THREAD

**Discussion by apastuszak** [rern/rAudio#899] [Bug Reports]
My NAs used to auto-mount when I rebooted my pi. Now I need to go into system settings and remount it manually. Anyone else seeing this?

**Comment by kw-sudo** [comment]
...same here. Occasionally I would test some fstab config parameters.

RasPi 3B+, rAudio-1 Release i20220415

Regards,
Oliver

[edit]: the settings make no difference, e.g

> x-systemd.automount,noauto,retry=0,x-systemd.idle-timeout=60,nolock,rw,timeo=2,retrans=1,x-systemd.device-timeout=2

Switched back to auto generated fstab

**Comment by ceejayemm** [comment]
What type of share is your NAS providing CIFS (Samba) or NFS ?  How do you address your NAS - via IP and Name ?  If the latter try switching to its direct IP address. I had a long discussion with Rern a while ago about the merits of IP vs Name and although I didn't want to do it I switched to IP Number for my NFS shares (provided by OMV v5.x) and all rAudio-1 network accesses and I have never had a problem since with losing share connections.

Chris

**Reply by kw-sudo** [reply]
I have a Synology DS716+II with the latest DRM version: 7.0.1-42218 Update 3. rAudio-1 access is via CIFS with a fixed IP.

I‘m not a Linux pro but I‘ve managed a ‚clean bitperfect‘ ArchLinuxARM install on my Odroid C1+ with mpd and lighttpd. fstab was nearly identical to rAudio-1 and with this remount after a reboot works fine. 

rAudio works as a network player. NAS (media server) —> rAudio (RPi3B+, LAN) —> amp (USB, integrated DAC, bitferfect). No renderers, no GPIO devices.

**Comment by cmh714** [comment]
Not having any issues myself. However I use nfs. if your using wifi, maybe test with a wire and see if maybe the network isnt ready when the mount occurs. just some guessing and some thoughts. GL!

**Comment by kw-sudo** [comment]
I think I got it for me!

First I took a closer look at the _settings/system/storage_...and pushed the question mark button.

![mount](https://user-images.githubusercontent.com/104136223/164687866-af0880c1-2d79-4247-a875-636129df0e37.jpg)

Until now I didn't give my NAS a separate name just set the music directory and the user/password. Now I followed the pic's advise, made a directory and adopted the path.

Second: fstab...i put in SMB version to vers=3.0 (Synology NAS is capable to min=1.0, max=3.0)

Two reboots and directory is remount. I'll keep an eye on that as a precaution.
